src/js/storyline.js

```diff
@@ -130,6 +152,27 @@ Storyline.prototype = {
   },
   positionSlider: function(slider) {
     this.elem.appendChild(slider.elem);
+  },
+  attachClickHandler: function(div, targetType) {
+    storyline.trackEvent('click', targetType)
```

Comment:
we should eliminate all references to a global `storyline` in favor of `this` (or `this.storyline` for components)